### ISSUE TITLE
Cannot attach softevent

### ISSUE THREAD

**Issue by rexrock** [cloudflare/ebpf_exporter#72]
There is a coding error in the PerfEvent struct：

```diff
diff --git a/config/config.go b/config/config.go
index 9c2a7a9..38ca87c 100644
--- a/config/config.go
+++ b/config/config.go
@@ -20,7 +20,7 @@ type Program struct {

 // PerfEvent describes perf_event to attach to
 type PerfEvent struct {
-       Type            int    `yaml:"string"`
+       Type            int    `yaml:"type"`
        Name            int    `yaml:"name"`
        Target          string `yaml:"target"`
        SamplePeriod    int    `yaml:"sample_period"`
```

**Comment by bobrik** [comment]
Indeed! In the examples we only use `HARDWARE` type, which conveniently happens to be zero, so it works as expected there.

Do you want to make a PR yourself to correct this?

**Comment by rexrock** [comment]
OK, I've done it.

**Comment by bobrik** [comment]
Fixed in #73.
